src/main/java/org/fundacion/jala/converter/core/javacompiler/JavaCompiler.java

```diff
@@ -79,4 +80,45 @@ public String javaCompiler(final JavaParameter newJavaParameter) throws Compiler
     public Result getResult() {
         return result;
     }
+
+    /**
+     * Compiles a java file.
```

Comment:
It would be better "Compiles a project java file."

Reply:
It is solved.